metrics.py

```diff
@@ -152,7 +152,7 @@ def get_stats(self):
                 errors = plugins_metadata['errors']
                 first_failed = sorted(errors.keys())[0]
                 plugins['failed_plugin'] = first_failed
-                plugins['exception'] = errors[first_failed].split("(")[0]
+                plugins['exception'] = errors[first_failed].split("(")[0].replace(',', ' ')
```

Comment:
Pretty sure it does support quoting, at least the pandas implementation does. Using `json.dumps()` also means we get quote-escaping 'for free'.